File: original-java-game/src/enemies/NormalEnemy.java
package enemies;

import source.Board;

public class NormalEnemy extends Enemy {
	
	public NormalEnemy(Board gui, int xs, int ys, int nivå) {
		super(gui.getBilder().getPeasant(), gui, xs, ys, gui.getLevel(), nivå);
	}

	@Override
	protected void setStats() {
		maxhealth = 100 + 100 * healthlevel;
		maxspeed = 1.2;
		damage = 1;
	}
}//END

File: src/entities/enemies/NormalEnemy.ts
import { Enemy } from './Enemy';


export class NormalEnemy extends Enemy {
    protected initStats(): void {
        this.maxHealth = 100 + 100 * this.healthLevel;
        this.health = this.maxHealth;
        this.maxSpeed = 60;
        this.speed = this.maxSpeed;
    }

    getSpriteName(): string {
        return 'peasant';
    }

}
